server/src/main/java/io/pravega/schemaregistry/service/SchemaRegistryService.java

```diff
@@ -264,7 +264,7 @@ public SchemaRegistryService(SchemaStore store, ScheduledExecutorService executo
         Preconditions.checkArgument(group != null);
```

Comment:
Can namespace be null? Why don't we do a null check for namespace here?

Reply:
namespace is not a mandatory parameter

Reply:
If namespace is optional would it be correct to base the "type" generation on an optional parameter?
What exactly does "type" signify for a schema?